apps/index.html

```diff
@@ -305,6 +305,22 @@
       </div>
 
       <div class="app">
+        <button onclick="install('118d77fa72eba6a685cb493bffb18fae?url=http://github.com/libreboard/libreboard/releases/download/v0.7.1/libreboard.spk')">Install »</button>
+        <h3>LibreBoard</h3>
+        <div class="app-details">
+          Originally by: <a href="https://github.com/yasaricli">Yaşar İçli</a> & <a href="https://github.com/mquandalle">Maxime Quandalle</a><br>
+          License: MIT<br>
+          Code: <a href="https://github.com/libreboard/libreboard">On GitHub</a><br>
+          Updated: Jan 11 2015
+        </div>
+        <p>A “kaban” board similar to Trello that let you organize things in cards, and cards in
```

Comment:
I think "kanban" is spelled with one more "n" character than what you wrote here.